.github/workflows/build-artifacts.yml

```diff
@@ -54,61 +74,75 @@ jobs:
       dry-run: false
       oidc-provider-name: database-gh-aerospike
       oidc-audience: database-gh-aerospike
-      jf-build-name: ${{ github.event.repository.name }}
+      jf-build-name: "aerospike-asadm-${{ needs.get-version.outputs.VERSION }}"
+
+  package-built-artifacts:
```

Comment:
I don't think this is actually packaging anything. It is just coalescing the matrix into one upload.